packages/shopify-codemod/transforms/remove-empty-catch-blocks.js

```diff
@@ -0,0 +1,16 @@
+export default function removeEmptyCatchBlocks({source}, {jscodeshift: j}, {printOptions = {}}) {
+
+  return j(source)
+    .find(j.TryStatement, {
+      type: 'TryStatement',
+      handler: {
+        type: 'CatchClause',
```

Comment:
There's a lot of those, we can do it later (we can even write a quick transform to do it for us :P)